README.md

```diff
@@ -37,38 +38,38 @@ You can specify a glob pattern as the value for schema.json which is the file na
 
 The following tasks can be done using VSCode:
 
-  a) Auto completion (Ctrl + Space):
-    Auto completes on all commands.
+a) Auto completion (Ctrl + Space):
```

Comment:
Not render as written: https://github.com/jenkinsci/jcasc-vscode-extension#user-guide

Perhaps best rewritten but better follow up in a separate PR.